.gitpod.Dockerfile

```diff
@@ -2,12 +2,3 @@ FROM gitpod/workspace-python
 
 RUN pyenv install 3.9.13 \
     && pyenv global 3.9.13
-
```

Comment:
Since we have record of the original dockerfile in the preAPI repo and in the core AaC, fully removing from here should be okay.